run-rsinstallpackagebuilder/src/pkgbuilder.js

```diff
@@ -0,0 +1,33 @@
+const os = require('os');
+const { loadTool } = require('../../utils');
+
+const getBinaryName = async () => {
+  let binaryName = '';
+
+  if (os.platform() !== 'win32') {
+    binaryName = 'InstallerPackageBuilder.Core.Console';
+  } else {
+    binaryName = 'InstallerPackageBuilder.Core.Console.exe';
+  }
+
+  return binaryName;
+};
+
+const getBuilder = async (appName, binaryVersion) => {
+  const binaryName = getBinaryName();
+  const downloadUrlNexus = `https://repo.extendaretail.com/repository/raw-hosted/${appName}/${binaryVersion}/${binaryName}`;
```

Comment:
This URL is probably not accessible without Nexus credentials within the URL itself.

Reply:
```
// process.env['NEXUS_USERNAME'] = '';
// process.env['NEXUS_PASSWORD'] = '';
```
Handled in the build of the action, for debug purposes is is just commented out in the tests.